inline-r/tests/tests.hs

```diff
@@ -105,4 +112,18 @@ tests = testGroup "Unit tests"
 main :: IO ()
 main = do
     _ <- R.initialize R.defaultConfig
-    defaultMain tests
+    argv <- getArgs
+    -- Assume gctorture() step size of 1.
+    let tortureCLI = "1" <$ find (== "--torture") argv
+    tortureEnv <- lookupEnv "R_GCTORTURE"
+    torture <- case tortureCLI <|> tortureEnv of
+      Nothing -> return False
+      Just x -> do
+        -- gctorture turned on. So assume moreover --enable-strict-barrier.
+        let step = read x :: Int32
+        putStrLn "WARNING: gctorture() turned on.\n\
+                 \    Tests will fail if R not compiled with --enable-strict-barrier."
```

Comment:
Nope - it's claiming you'll get false negatives. If you pass `--torture` but didn't compile R with `--enable-strict-barrier`, then the last test in `Test.GC` will (likely) fail.